dtfs-central-api/api-tests/v1/tfm/facilities/tfm-amendments-get.api-test.js

```diff
@@ -0,0 +1,136 @@
+const wipeDB = require('../../../wipeDB');
+const app = require('../../../../src/createApp');
+const api = require('../../../api')(app);
+const CONSTANTS = require('../../../../src/constants');
+const { MOCK_DEAL } = require('../../mocks/mock-data');
+const aDeal = require('../../deal-builder');
+
+const mockUser = {
+  _id: '123456789',
+  username: 'temp',
+  roles: [],
+  bank: {
+    id: '956',
+    name: 'Barclays Bank',
+  },
+};
+
+const newFacility = {
+  type: 'Bond',
+  dealId: MOCK_DEAL.DEAL_ID,
+};
+
+const newDeal = aDeal({
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS,
+  additionalRefName: 'mock name',
+  bankInternalRefName: 'mock id',
+  editedBy: [],
+  eligibility: {
+    status: 'Not started',
+    criteria: [{}],
+  },
+});
+const createDeal = async () => {
+  const { body } = await api.post({ deal: newDeal, user: mockUser }).to('/v1/portal/deals');
+  return body;
+};
+
+describe('/v1/tfm/facilities/:id/amendments', () => {
+  let dealId;
+
+  beforeAll(async () => {
+    await wipeDB.wipe(['tfm-deals']);
+    await wipeDB.wipe(['tfm-facilities']);
```

Comment:
we can simplify this by saying 
`await wipeDB.wipe(['tfm-facilities', 'tfm-deals']);`

Reply:
done